src/LearnositySdk/Utils/Uuid.php

```diff
@@ -2,11 +2,13 @@
 
 namespace LearnositySdk\Utils;
 
+require_once __DIR__ . '/../../../vendor/paragonie/random_compat/lib/random.php';
```

Comment:
I assume this is the polyfill, and it gives us `random_bytes`. Could we add a comment to this effect here? Also, it would be good to list what latest version of PHP it is needed for.

Reply:
(ok, PHP7)